source/OutfitterPanel.cpp

```diff
@@ -151,7 +151,15 @@ void OutfitterPanel::DrawItem(const string &name, const Point &point, int scroll
 	}
 	if(player.Cargo().Get(outfit))
 	{
-		string count = "in cargo: " + to_string(player.Cargo().Get(outfit));
+		string count;
+		if(!outfitter.Has(outfit) and player.Stock(outfit) > 0)
```

Comment:
Would you want to use && instead of "and"? I don't see "and" being used like this anywhere else.